pycycling/sterzo.py

```diff
@@ -22,10 +23,19 @@ async def enable_steering_measurement_notifications(self):
         await self._activate_steering_measurements()
 
     async def _activate_steering_measurements(self):
-        with importlib.resources.files(__package__).joinpath('data/sterzo-challenge-codes.dat').open('rb') as fp:
-            fp.seek(self._latest_challenge * 2, 1)
-            code_1 = int.from_bytes(fp.read(1), 'little')
-            code_2 = int.from_bytes(fp.read(1), 'little')
+        # moved .dat file out of 'data' because accessing directories not possible for importlib.resources.path
+        # importlib.resources.path is deprecated since 3.11
+        if sys.version_info >= (3,11):
+            with importlib.resources.files(__package__).joinpath('sterzo-challenge-codes.dat').open('rb') as fp:
+                fp.seek(self._latest_challenge * 2, 1)
+                code_1 = int.from_bytes(fp.read(1), 'little')
+                code_2 = int.from_bytes(fp.read(1), 'little')
+        else: # legacy support < 3.9
+            with importlib.resources.path(__package__, 'sterzo-challenge-codes.dat') as file_path:
```

Comment:
Is there a reason not to use `importlib.resources.open_binary`?

Reply:
Will be using that instead.